app/src/main/java/com/example/myapplication/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.myapplication.mvp.model
+
+import com.example.myapplication.utils.EMPTY_STRING
+import com.example.myapplication.utils.NULL_RESULT
+
+class CalculatorModel {
+    private var operator1: String = EMPTY_STRING
```

Comment:
i forget to mention. try to avoid digits in variable names. 
try operatorOne instead